packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -180,7 +180,7 @@ export const MafsGraph = (props: MafsGraphProps) => {
                         interactiveElementsDescription &&
                             interactiveElementsDescriptionId,
                         isUnlimitedGraphState(state) &&
-                            "unlimited-graph-keyboard-prompt",
```

Comment:
This wasn't doing anything, since the actual prompt's ID is `unlimitedGraphKeyboardPromptId`, which includes the unique ID from `React.useId()`. Fixed it here so it actually gets the correct ID.